mungegithub/mungers/publisher.go

```diff
@@ -134,31 +151,33 @@ func clone(dst string, dstURL string) error {
 // construct checks out the source repo, copy the contents to the destination,
 // returns a commit message snippet and error.
 // this function is only used when publishing client-go.
-func construct(base, org string, src, dst coordinate) (string, error) {
+func (p *PublisherMunger) construct(src, dst coordinate) (string, error) {
+	base := p.baseDir
+	org := p.githubConfig.Org
 	srcRepoRoot := filepath.Join(base, src.repo)
 	srcDir := filepath.Join(base, src.repo, src.dir)
 	dstRepoRoot := filepath.Join(base, goWorkspaceStructure, dst.repo)
 	curDir, err := os.Getwd()
 	if err != nil {
-		glog.Infof("Getwd failed")
+		p.plog.Infof("Getwd failed")
```

Comment:
Error is logged when the function returned. I found the error sometimes didn't contain enough information, so added Infof here and belowl